pkg/nsx/client.go

```diff
@@ -83,5 +99,29 @@ func GetClient(cf *config.NSXOperatorConfig) *Client {
 		SecurityClient: securityClient,
 		RuleClient:     ruleClient,
 		NSXChecker:     *nsxChecker,
+		NSXVerChecker:  *nsxVersionChecker,
+	}
+}
+
+func (client *Client) CheckNsxVersion() bool {
```

Comment:
how about add a parameter minVersion for CheckNsxVersion()?  using a global variable by nsxVersion.featureSupported maybe not easy to change later if each controller has its own minVersion